apps/todolist-mariadb-go/test/test.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+from datetime import datetime
+import json
+import requests
+
+
+
+def updateToDo(base_url, id, completed):
+  """Update data to the todo application
+
+  Args:
+    base_url: url
+    item_dict: dict of todo item
+    completed: bool
+
+  Returns:
```

Comment:
This doesn't Return as it's in the docstring. Func returns True or False only. The response is being sent with the endpoint/data=data, but it's not return from the funct.